lib/screens/package_details/package_details_bloc.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:async';
+
+import 'package:bloc/bloc.dart';
+import 'package:flutter/foundation.dart';
+import 'package:pub_client/pub_client.dart';
+import 'package:tavern/screens/bloc.dart';
+import 'package:tavern/screens/package_details/package_details_event.dart';
+import 'package:tavern/src/cache.dart';
+
+class PackageDetailsBloc
```

Comment:
you can make this a HydratedBloc as well and you don't need the PackageRepository to have it's own cache.

Reply:
I don't currently have any state to serialize the cache is going to be session based

Reply:
what do you mean? The PackageDetailsBloc has to have state

Reply:
I meant that I don't have anything I want to persist to the device, so I don't feel that I need a hydrated bloc